psydac/linalg/stencil.py

```diff
@@ -597,6 +627,7 @@ def ghost_regions_in_sync(self, value):
 
     # ...
     # TODO: maybe change name to 'exchange'
+
     def update_ghost_regions(self):
```

Comment:
A useless blank line was added. Please remove it

Reply:
[52e87de](https://github.com/pyccel/psydac/pull/273/commits/52e87de15aa55a0093db0a85d90077e1c0e51551)